ACP2E-994: [Documentation] Add notice that Admin, Bearer and other types of tokens are not supported to skip providing of HTTP X-Captcha header. (#20)
AdobeDocs/commerce-webapi: src/pages/rest/use-rest/protected-endpoints.md

If CAPTCHA or reCAPTCHA is enabled on pages requiring shopper input, then in most cases, the corresponding endpoints that send requests to the Adobe Commerce server must include an HTTP header that contains a value entered by the shopper (for CAPTCHA) or generated by the Google API (for reCAPTCHA). However, if you specify an integration authorization token in the header of the endpoint, then you do not supply a header specific to CAPTCHA or reCAPTCHA.

> HTTP `X-Captcha` and `X-ReCaptcha` headers:

Reviewer: ```suggestion
The HTTP `X-Captcha` and `X-ReCaptcha` headers:
```

> *  Cannot be received by automated script or non-UI API call. They are captured and returned by UI Web form only.

Reviewer: ```suggestion
*  Cannot be received by an automated script or a non-UI API call. They are captured and returned by the UI Web form only.
```

> *  Are optional in protected mutation API calls with providing **_integration authorization_** tokens only. They cannot be skipped with Admin and Bearer tokens.

Reviewer: ```suggestion
*  Is optional in protected mutation API calls that provide **_integration authorization_** tokens only. It cannot be skipped when you provide an Admin or Bearer token.
```

> *  Is optional in protected mutation API calls that provide **_integration authorization_** tokens only. It cannot be skipped when you provide an Admin or Bearer token.

Reviewer: ```suggestion
*  Are optional in protected mutation API calls that provide **_integration authorization_** tokens only. They cannot be skipped when you provide an Admin or Bearer token.
```

Author: Fixed.
